### PR TITLE
Introduce gnome-sound-recorder

### PR DESCRIPTION
**Pull request by gaurav1999**
Gnome-sound recorder is one the well maintained and active project from GNOME developers, it's  nice start for one wishing to get started with FOSS, might be a good idea to add it to this list.

Thanks.

### CODE REVIEW

**Comment by gaurav1999** [comment]
@bilelmoussaoui , would like to know your view on the tags and description used. 

Thanks.

**Comment by bilelmoussaoui** [comment]
@gaurav1999 Perfect. Thanks!

File: _data/gnome-sound-recorder.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- javascript
+- gtk3
+- gnome
+- gui
+
+
+
+upforgrabs:
+  name: newcomers
+  link: https://gitlab.gnome.org/GNOME/gnome-sound-recorder/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=4.%20Newcomers
```
Comment (ritwik12) [inline _data/gnome-sound-recorder.yml]: @gaurav1999 Hey, I don't see any open issues with this label there, It is good to have some open tasks/ issues for contributors to work on. Please add some tasks or use some other label.

File: _data/gnome-sound-recorder.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+site: https://wiki.gnome.org/Apps/SoundRecorder
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- javascript
+- gtk3
+- gnome
+- gui
+
+
+
```
Comment (ritwik12) [inline _data/gnome-sound-recorder.yml]: Please remove these extra lines.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/gnome-sound-recorder.yml (hunk 1 above)
Comment (bilelmoussaoui) [reply _data/gnome-sound-recorder.yml]: I have added two for now. I will look into adding more tomorrow. Thanks!

Comment (ritwik12) [reply _data/gnome-sound-recorder.yml]: Thanks

File: _data/gnome-sound-recorder.yml (hunk 2 above)
Comment (ritwik12) [reply _data/gnome-sound-recorder.yml]: @gaurav1999

**Review by shiftkey — CHANGES_REQUESTED** [review]
@gaurav1999 this project needs to be located in the `_data/projects/` directory for it to detected (it's currently only under `_data/`)

File: _data/gnome-sound-recorder.yml (hunk 2 above)
Comment (gaurav1999) [reply _data/gnome-sound-recorder.yml]: Should be fine now, sorry missed to add these lines while raising a PR that time, thanks for review.

**Comment by gaurav1999** [comment]
@shiftkey , yes missed it because of my ide :/ , updated the location of file, I guess should be fine now. 
Thanks for review.

**Review by shiftkey — APPROVED** [review]

**Review by ritwik12 — APPROVED** [review]
lgtm!!

**Comment by ritwik12** [comment]
@gaurav1999 Thanks!!

**Pull request merged**